### PR TITLE
Generate a license directory that contains licenses missing from the installers

### PR DESCRIPTION
**Pull request by pmaslana**
### Purpose:
PIP and NPM packages are being included in some chia installation patterns without their licenses. 
This creates a tree of directories that contains a license for the packages.

### Current Behavior:
No

### New Behavior:
N/A

### Testing Notes:
Verify that the directory exists.

### CODE REVIEW

File: build_scripts/build_license_directory.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+for i in "${licenses_array[@]}"; do
+    dirname="licenses/$(dirname "$i" | awk -F'/' '{print $NF}')"
+    mkdir -p "$dirname"
+    echo "$dirname"
+    cp "$i" "$dirname"
+done
+
+mv licenses/ ../build_scripts/dist/daemon
+cd ../build_scripts || 'exit" or "cd' || return
```
Comment (Starttoaster) [inline build_scripts/build_license_directory.sh:29]: Having trouble understanding the quotation marks in this. What does `'exit" or "cd'` _do_?

File: build_scripts/build_win_license_dir.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# PULL IN LICENSES USING NPM - LICENSE CHECKER
+npm install -g license-checker
+
+cd ../chia-blockchain-gui || 'exit" or "cd' ... || return
```
Comment (Starttoaster) [inline build_scripts/build_win_license_dir.sh:6]: Same thing here about interesting quotations but with the addition of an ellipses

File: build_scripts/build_win_license_dir.sh [hunk 3]
```diff
@@ -26,7 +26,7 @@ for i in "${licenses_array[@]}"; do
 done
 
 mv licenses/ ../build_scripts/dist/daemon
-cd ../build_scripts || 'exit" or "cd' || return
+cd ../build_scripts || || exit 1
```
Comment (Starttoaster) [inline build_scripts/build_win_license_dir.sh:29]: This shouldn't have two sets of `||`

File: build_scripts/build_license_directory.sh (hunk 1 above)
Comment (pmaslana) [reply build_scripts/build_license_directory.sh]: These are a carryover. I changed it to "exit 1".

File: build_scripts/build_win_license_dir.sh (hunk 2 above)
Comment (pmaslana) [reply build_scripts/build_win_license_dir.sh]: These are a carryover. I changed it to "exit 1".

File: build_scripts/build_license_directory.sh [hunk 4]
```diff
@@ -26,7 +26,7 @@ for i in "${licenses_array[@]}"; do
 done
 
 mv licenses/ ../build_scripts/dist/daemon
-cd ../build_scripts || 'exit" or "cd' || return
+cd ../build_scripts || || exit 1
```
Comment (Starttoaster) [inline build_scripts/build_license_directory.sh:29]: This shouldn't have two sets of `||`

Comment (pmaslana) [reply build_scripts/build_license_directory.sh]: Fixed

File: build_scripts/build_win_license_dir.sh (hunk 3 above)
Comment (pmaslana) [reply build_scripts/build_win_license_dir.sh]: Fixed

**Review by Starttoaster — APPROVED** [review]
LGTM

**Review by emlowe — APPROVED** [review]

**Pull request merged**